config/testgrids/config.yaml

```diff
@@ -2585,559 +2071,6 @@ dashboards:
-  - name: integ-galley-local-presubmit-tests
-    test_group_name: integ-galley-local-presubmit-tests-release-1.2
-  - name: integ-galley-k8s-presubmit-tests
-    test_group_name: integ-galley-k8s-presubmit-tests-release-1.2
-  - name: integ-mixer-local-presubmit-tests
-    test_group_name: integ-mixer-local-presubmit-tests-release-1.2
-  - name: integ-mixer-k8s-presubmit-tests
-    test_group_name: integ-mixer-k8s-presubmit-tests-release-1.2
-  - name: integ-pilot-local-presubmit-tests
-    test_group_name: integ-pilot-local-presubmit-tests-release-1.2
-  - name: integ-pilot-k8s-presubmit-tests
-    test_group_name: integ-pilot-k8s-presubmit-tests-release-1.2
-  - name: integ-security-local-presubmit-tests
-    test_group_name: integ-security-local-presubmit-tests-release-1.2
-  - name: integ-security-k8s-presubmit-tests
-    test_group_name: integ-security-k8s-presubmit-tests-release-1.2
-  - name: pilot-e2e-envoyv2-v1alpha3
-    test_group_name: istio-pilot-e2e-envoyv2-v1alpha3-presubmit-release-1.2
-  - name: e2e-mixer-no_auth
-    test_group_name: istio-e2e-mixer-no_auth-presubmit-release-1.2
-  - name: e2e-dashboard
-    test_group_name: istio-e2e-dashboard-presubmit-release-1.2
-  - name: e2e-simpleTests
-    test_group_name: istio-e2e-simpleTests-presubmit-release-1.2
-  - name: e2e-bookInfoTests-envoyv2-v1alpha3
-    test_group_name: istio-e2e-bookInfoTests-envoyv2-v1alpha3-presubmit-release-1.2
-  - name: e2e-bookInfoTests-trustdomain
-    test_group_name: istio-e2e-bookInfoTests-trustdomain-presubmit-release-1.2
-  - name: circleci-e2e-simple
-    test_group_name: istio-circleci-e2e-simple-presubmit-release-1.2
-  - name: circleci-e2e-dashboard
-    test_group_name: istio-circleci-e2e-dashboard-presubmit-release-1.2
-  - name: circleci-e2e-mixer-noauth-v1alpha3-v2
-    test_group_name: istio-circleci-e2e-mixer-noauth-v1alpha3-v2-presubmit-release-1.2
-  - name: circleci-e2e-galley
-    test_group_name: istio-circleci-e2e-galley-presubmit-release-1.2
-  - name: circleci-e2e-pilot-cloudfoundry-v1alpha3-v2
-    test_group_name: istio-circleci-e2e-pilot-cloudfoundry-v1alpha3-v2-presubmit-release-1.2
-  - name: circleci-e2e-pilot-auth-v1alpha3-v2
-    test_group_name: istio-circleci-e2e-pilot-auth-v1alpha3-v2-presubmit-release-1.2
-  - name: circleci-e2e-pilot-noauth-v1alpha3-v2
-    test_group_name: istio-circleci-e2e-pilot-noauth-v1alpha3-v2-presubmit-release-1.2
-  - name: circleci-unit-tests
-    test_group_name: istio-circleci-test-presubmit-release-1.2
-  - name: circleci-codecov-tests
-    test_group_name: istio-circleci-codecov-presubmit-release-1.2
-  - name: circleci-racetest
-    test_group_name: istio-circleci-racetest-presubmit-release-1.2
-  - name: circleci-integration-local
-    test_group_name: istio-circleci-test-integration-local-presubmit-release-1.2
-  - name: circleci-integration-kubernetes
-    test_group_name: istio-circleci-test-integration-kubernetes-presubmit-release-1.2
-
-- name: istio-postsubmits
```

Comment:
Same issue here; there's an `istio-postsubmits-release-1.2` and an `istio-postsubmits-master`, but not `istio-postsubmits`